src/components/AppLayout/menuStructure.ts

```diff
@@ -53,9 +53,10 @@ function useMenuStructure(
     navigate(menuItem.url, { resetScroll: true });
   };
 
-  const extenstionHeaderItem = {
+  const appExtensionsHeaderItem = {
     id: "extensions",
-    label: intl.formatMessage(commonMessages.extensions)
+    ariaLabel: "apps",
+    label: intl.formatMessage(sectionNames.apps)
```

Comment:
Are we sure it's a good idea to reuse the section name translation as the header? Will they ever need to be translated differently?

Reply:
fixed 👍